vmdb/lib/miq_automation_engine/service_models/miq_ae_service_orchestration_stack.rb

```diff
@@ -28,5 +28,13 @@ def remove_from_vmdb
       @object = nil
       true
     end
+
+    def raw_exists?
+      status, reason = raw_status
+      status.nil? || status.downcase == 'delete_complete' ? false : true
+    rescue => err
+      return false if err.to_s =~ /[S|s]tack.+does not exist/
```

Comment:
I'm worried about checking the message of the error.  Isn't there a specific exception that we can rescue from so that we don't have to rely on what the message says?